### ISSUE TITLE
Response type of `text/javascript`

### ISSUE THREAD

**Issue by chadfawcett** [fbsamples/messenger-platform-samples#28]
When posting to the messages api endpoint in the `callSendAPI` method, it is being sent as `application/json` but the response type is `text/javascript`. https://github.com/fbsamples/messenger-platform-samples/blob/830abab21750d8441533ff123159dfa47252a0a6/node/app.js#L813

Is this the desired behaviour? I am attempting to use `superagent` instead of the `request` module to make my requests. It will not parse the response body unless the type is correct. In my case I am attempting to parse json, so it won't parse the response to body as the type isn't `application/json`. 

Basically I'm just wondering if the type is correct. I couldn't find anywhere to Ask Questions/Report Issues on the API itself.

**Comment by chadfawcett** [comment]
Sorry. I tried setting the `Accept` header on my request and thought it didn't work. Turned out my source had an issue.

Setting the header `Accept: application/json` results in the response being in the desired type.
